src/ticketmap/index.tsx

```diff
@@ -216,10 +220,16 @@ export default class TicketMap extends Component<Props & DefaultProps, State> {
 
   deselectSection = (section?: string) => {
     if (!section) {
-      this.state.selectedSections.clear()
-      return this.updateMap()
+      this.clearSelection()
```

Comment:
The `else` doesn't seem necessary here
```javascript
if (!section) {
  return this.clearSelection()
}
return this.toggleSectionSelect(section, false)
```

Reply:
I'm not sure I follow... why would `deselectSection()` return the result of `clearSelection()`?

Reply:
Because we want to explicitly exit the function after `clearSelection()` runs, which means we could do
```javascript
this.clearSelection();
return;
```
However since they both return `void`, I see no harm in combining the statements. The goal here is to reduce code by explicitly exiting when we intend to exit, and by exiting early.

Reply:
Some info on the topic https://blog.codinghorror.com/flattening-arrow-code/

Reply:
The goal of that article is to convey some approaches to flattening deeply nested logic. I see no deeply nested logic here.